.github/ISSUE_TEMPLATE/release-checklist.yaml

```diff
@@ -46,6 +46,7 @@ body:
       - label: knative.dev/eventing-kogito
       - label: knative.dev/eventing-rabbitmq
       - label: knative.dev/sample-source
+      - label: knative.dev/client-pkg
       - label: knative.dev/client
```

Comment:
If `client` depends on `client-pkg`, this will need to be in the next wave

Reply:
I don't think that's the case... client [go.mod](https://github.com/knative/client/blob/df40f5a38991c7698b9b1382ee29d75209e114e8/go.mod#L23-L27) doesn't list client-pkg as a dependency, but would be good to confirm with @rhuss @dsimansk

Reply:
Even when it is not listed yet, it is supposed to become a dependency of the client. The main motivation for the client-pkg package is to avoid cyclic dependencies and move out shared code used by both the client and its plugins (like parsing common command line options).

So client-pkg should be in one wave before the client, and also should follow the same semantics as `pkg`, i.e. should be cut one week before release.

Reply:
That would mean that client-pkg would be cut before serving/eventing, but client-pkg also has severing/eventing in its go.mod ([eventing](https://github.com/knative/client-pkg/blob/e9619222f136000b674f8555f3e0b9ddd4709c1c/go.mod#L21) and [serving](https://github.com/knative/client-pkg/blob/e9619222f136000b674f8555f3e0b9ddd4709c1c/go.mod#L24) ). Just want to double check on that...

Reply:
That's a very good point that we need to clarify in the client WG: Is client-pkg allowed to reference also serving or eventing parts ? cc @dsimansk 

Depending on the answer:

* If "yes", then client-pkg should be created on the usual release train, after all, serving/eventing but before the client and any plugin.
* If "no", it can be cut 1 week before together with pkg, but after pkg (supposed the client-pkg reuses something from pkg)

Actually, to keep the flexibility it would make sense to create client-pkg along the usual release trains (so the first "yes" option).

Reply:
updating to more explicitly separate client-pkg from client in the timeline